scripts/parse-icons.js

```diff
@@ -48,7 +48,7 @@ const getPath = nodes => nodes.children
 
 const getViewBox = nodes => nodes.properties.viewBox
 
-const removePrefix = str => str.replace(/^ic\-/, '')
+const removePrefix = str => str.replace(/^ic\-/, '') // eslint-disable-line no-useless-escape
```

Comment:
Shouldn't the regex just be updated rather than disabling the lint?

Reply:
Ideally, do you think removing the \` before the `-` should fix it?